Makefile

```diff
@@ -85,6 +85,10 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 i18n-lint:
 	@echo "$(JS_FILES)" | sed 's/\([^ ]*\/test\/[^ ]* *\)//g' | xargs -n1 $(I18NLINT)
 
+# Skip files that are auto-generated
+mixedindentlint: node_modules/mixedindentlint
+	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs $(NODE_BIN)/mixedindentlint --ignore-comments --exclude="client/config/index.js" --exclude="shared/components/gridicon/index.jsx"
```

Comment:
>unless this comment was triggered by some other, related issue which could serve the purpose?

Nah, not particularly. Happened to stumble upon it in the course of reviewing #3793.